console/src/main/java/org/georchestra/console/ws/backoffice/roles/RolesController.java

```diff
@@ -519,12 +523,21 @@ private Role createRoleFromRequestBody(ServletInputStream is) throws IOException
 				throw new IllegalArgumentException(RoleSchema.COMMON_NAME_KEY + " is required" );
 			}
 
+			// Capitalize role name and check format
+			commonName = commonName.toUpperCase();
+			Pattern p = Pattern.compile("[A-Z0-9_]+");
```

Comment:
Looks like:
 - `_TRUITE`
 - `TRUITE__VELUE`
 - `TRUITE_`

will be validated but shouldn't. — or maybe it's out of this PR scope? —

Reply:
You're right @tonio, I forgot these ones...